jobs/build.Jenkinsfile

```diff
@@ -23,7 +22,7 @@ properties([
              defaultValue: '',
              trim: true),
       string(name: 'ADDITIONAL_ARCHES',
-             description: 'Override default additional target architectures (space-separated)',
+             description: "Override additional architectures (space-separated). Supported: ${pipeutils.get_supported_additional_arches().join(' ')}",
```

Comment:
optional: long line: any chance we can break this to a second line? same for below

Reply:
Fixed!